README.md

```diff
@@ -158,6 +158,48 @@ Just want to log violations in production:
 ActionController::Parameters.action_on_invalid_parameters = :log
 ```
 
+## ControllerSupport::ParameterWhitelist
+
+When included into a controller, this module forces the developer
+to explicitly whitelist which parameters are passed to any methods.
+
+Examples:
+
+```ruby
+class TestController < ApplicationController
+  include StrongerParameters::ControllerSupport::ParameterWhitelist
+
+  allow_parameters :show, forum_id: Parameter.integer
+  allow_parameters :create, topic: { forum: { :id: Parameter.integer } }
+  allow_parameters :index, {} # no parameters allowed
+  allow_parameters :update, :anything # all parameters allowed, should only be used when migrating old controllers/actions
+end
+```
+
```

Comment:
Header should be it's own sub-topic `Notifying users about unpermitted parameters`